pkg/cmd/utils/gen-release-notes/generate.go

```diff
@@ -0,0 +1,193 @@
+// Copyright (C) 2021 ScyllaDB
+
+package gen_release_notes
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/object"
+	"github.com/go-git/go-git/v5/plumbing/storer"
+	"github.com/google/go-github/v35/github"
+	"github.com/scylladb/scylla-operator/pkg/cmdutil"
+	"github.com/scylladb/scylla-operator/pkg/genericclioptions"
+	"github.com/spf13/cobra"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+const (
+	EnvVarPrefix = "SCYLLA_OPERATOR_GEN_RELEASE_NOTES_"
+)
+
+func NewGenGitReleaseNotesCommand(ctx context.Context, streams genericclioptions.IOStreams) *cobra.Command {
+	o := NewGitGenerateOptions(streams)
+
+	cmd := &cobra.Command{
+		Short: "Generates release notes based on merge commits from local git repository between two tags.",
+		Long: templates.LongDesc(`
+		Scylla Operator Release Notes
+
+		This command generates release notes based on merge commits from local git repository between two tags.
+		`),
+
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			return cmdutil.ReadFlagsFromEnv(EnvVarPrefix, cmd)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := o.Validate(); err != nil {
+				return err
+			}
+
+			if err := o.Run(ctx); err != nil {
+				return err
+			}
+
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVar(&o.ReleaseName, "release-name", o.ReleaseName, "Name of the release.")
+	cmd.Flags().StringVar(&o.RepositoryPath, "repository-path", o.RepositoryPath, "Path to the git repository.")
+	cmd.Flags().StringVar(&o.RefStart, "ref-start", o.RefStart, "PRs merged after before this ref won't be considered.")
```

Comment:
The comment should answer the question of "What is ref-start?". "First commit reference." You can describe the behaviour after if needed.

Reply:
naming wise I'd prefer `start-ref` to `ref-start`, similarly you'd use `start-number`, not `number-start`